src/api/cristinApi.ts

```diff
@@ -163,3 +164,20 @@ export const fetchProfilePicture = async (cristinId: string) => {
   });
   return fetchProfilePictureResponse.data;
 };
+
+export const searchForKeywords = async (results: number, page: number, query?: string) => {
+  const searchParams = new URLSearchParams();
+  if (query) {
+    searchParams.set('query', query);
+  }
+  searchParams.set('results', results.toString());
+  searchParams.set('page', page.toString());
+
+  const queryContent = searchParams.toString();
+  const queryParams = queryContent ? `?${queryContent}` : '';
+
+  const fetchKeywordsResponse = await authenticatedApiRequest2<SearchResponse<Keywords>>({
```

Comment:
trenger vel ikke token for dette?
```suggestion
  const fetchKeywordsResponse = await apiRequest2<SearchResponse<Keywords>>({
```

Reply:
sikkerhetsrisiko å sende/eksponere token til tjenester som ikke trenger det